cmd/odo/odo.go

```diff
@@ -43,6 +43,13 @@ func main() {
 		}
 	}
 
+	// Override the logging level by the value (if set) by the ODO_LOG_LEVEL env
+	// The "-v" flag set on command line will take precedence over ODO_LOG_LEVEL env
+	v := flag.CommandLine.Lookup("v").Value.String()
+	if level, ok := os.LookupEnv("ODO_LOG_LEVEL"); ok && v == "0" {
+		_ = flag.CommandLine.Set("v", level)
```

Comment:
Should we check  if the error was returned or not?

Reply:
We can, if we want to take some corrective action.